web/src/index.css

```diff
@@ -490,3 +498,7 @@ html[data-theme='dark-mode'] .ui.table {
 .ui.attached.inverted.menu {
     border: 1px solid #555 !important;
 }
+
+.dimmed.dimmable > .ui.modals.dimmer.visible {
+    display: flex !important;
```

Comment:
Not sure I approve of this haha, but also not too fussed as it seems like we've been fairly liberal with !important in the past